Plugin with id 'com.gradle.build-scan' not found.

## Repro:
`git clone https://github.com/jaredsburrows/android-gif-example`
`cd android-gif-example`
`gradlew assembleDebug --scan`

## Example build:
https://travis-ci.org/jaredsburrows/android-gif-example/builds/313230232?utm_source=github_status&utm_medium=notification

## Error:
```
$ gradlew assembleDebug --scan
Download https://plugins.gradle.org/m2/com/gradle/build-scan-plugin/1.10.3/build-scan-plugin-1.10.3.pom
Download https://plugins.gradle.org/m2/com/gradle/build-scan-plugin/1.10.3/build-scan-plugin-1.10.3.jar

FAILURE: Build failed with an exception.

* Where:
Auto-applied by using --scan

* What went wrong:
Could not apply requested plugin [id: 'com.gradle.build-scan', version: '1.10.3', artifact: 'com.gradle:build-scan-plugin:1.10.3'] as it does not provide a plugin with id 'com.gradle.build-scan'. This is caused by an incorrect plugin implementation. Please contact the plugin author(s).
> Plugin with id 'com.gradle.build-scan' not found.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
```

--- a/gradle/dependencies.gradle
+++ b/gradle/dependencies.gradle
@@ -19,7 +19,7 @@ ext {
     libraries = [:]
 }
 
-versions.gradle_kotlin_dsl = '0.13.1'
+versions.gradle_kotlin_dsl = '0.13.2'
 
 versions.commons_io = 'commons-io:commons-io:2.2'
 

